composer.json

```diff
@@ -11,22 +11,22 @@
     ],
     "require": {
         "php": ">=8.0.5",
-        "bref/bref": "^1.3",
+        "bref/bref": "^1.3 || ^2.0",
         "clue/arguments": "^2.1",
         "guzzlehttp/psr7": "^1.8 || ^2.0",
-        "illuminate/contracts": "^8.33",
+        "illuminate/contracts": "^8.33 || ^9.0",
```

Comment:
Why Laravel 9 here?

Reply:
same here ...dependencies to symfony 6.x components